cvat/schema.yml

```diff
@@ -8338,9 +8338,6 @@ components:
         assignee:
           type: integer
           nullable: true
-        message:
-          type: string
-          minLength: 1
```

Comment:
This seems odd to me. Should we not allow editing issue messages? Why not?

Reply:
Because they must be edited via the first comment. The field doesn't work.